riskified-sdk/src/main/java/com/riskified/models/ShippingLine.java

```diff
@@ -51,4 +66,6 @@ public TaxLines getTaxLines() {
     public void setTaxLines(TaxLines taxLines) {
```

Comment:
this is a prior bug - TaxLines is misnamed and is actually a single instance of a Tax Line. The taxLines instance variables should be of type List<TaxLines>.